src/aics-image-viewer/components/ChannelsWidgetRow/styles.css

```diff
@@ -23,6 +23,19 @@
     .ant-checkbox-wrapper {
       color: var(--color-controlpanel-text);
     }
+
+    .ant-btn-text {
+      color: var(--color-button-icon-activated-text);
+      transition: color 0.3s;
+
+      &:hover {
+        color: var(--color-button-tertiary-hover-text) !important;
+      }
+    }
+
+    &.controls-closed .ant-btn-text:not(:focus-visible) {
```

Comment:
Any reason for having different focus and hover colors? It seems like you could use the same for both to me?

Reply:
The button still has its focus outline behavior